dotCMS/src/main/java/com/dotmarketing/factories/PublishFactory.java

```diff
@@ -167,7 +168,7 @@ public static boolean publishAsset(Inode webAsset, User user, boolean respectFro
 		if (webAsset instanceof WebAsset)
 		{
 			try {
-				WebAssetFactory.publishAsset((WebAsset) webAsset, user, isNewVersion);
+				WebAssetFactory.publishAsset((WebAsset) webAsset, user, isNewVersion); // todo: reviewing here
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)